src/app.py

```
@@ -1,105 +1,153 @@
+
 openai_api_key = ""
 
-with open('C:\\Users\\alper\\Desktop\\mervesultan_openai_key.txt', 'r') as file:
-    openai_api_key = file.read().strip()
+try:
+    with open('C:\\Users\\alper\\Desktop\\mervesultan_openai_key.txt', 'r') as file:
+        openai_api_key = file.read().strip()
+    logger.info("OpenAI API anahtarı başarıyla yüklendi.")
+except Exception as e:
+    logger.error(f"OpenAI API anahtarı yüklenemedi: {e}")
+    raise RuntimeError(f"OpenAI API anahtarı yüklenemedi: {e}")
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)